src/components/TitleHashButton.tsx

```diff
@@ -135,13 +137,22 @@ function Name({address, isValidator}: {address: string; isValidator: boolean}) {
           padding: "0.15rem 1rem 0.15rem 1rem",
         }}
       >
-        <Link
-          href={`https://www.aptosnames.com/name/${name}`}
-          target="_blank"
-          underline="none"
-        >
-          {`${name}.apt`}
-        </Link>
+        {isAns ? (
+          <Link
+            href={`https://www.aptosnames.com/name/${name}`}
+            target="_blank"
+            underline="none"
+          >
+            {name}
+          </Link>
+        ) : (
+          <Typography sx={{display: "flex", alignItems: "row", gap: 1}}>
+            <span>{name}</span>
+            <Tooltip title={"This is a verified address label."}>
+              <VerifiedOutlined fontSize="small" />
```

Comment:
Sorry to nitpick, the alignment was still slightly off. I think perhaps the line height of the font is causing it. This should address it. 

```suggestion
              <VerifiedOutlined fontSize="small" sx={{position: 'relative', top: 2}} />
```